File: 1174.py
import sys
sys.stdin = open('input.txt', 'r')
sys.stdout = open('output.txt', 'w')

chave = []
valor = []

tam = 0

for i in range (0, 100):
    x = int(input())
    if(x <= 10):
        chave.append(i)
        valor.append(x)
        tam+=1


for i in valor:

    if(chave[i] == tam):
        print(f"A[{chave[i]}] = {valor[i]}", end="")

    if(valor[i] <= 10):
        print(f"A[{chave[i]}] = {valor[i]}")
        

File: 1174.cpp
#include <bits/stdc++.h>

using namespace std;

int main (){

    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);

    vector<pair<int, float>> v;

    float x;

    for(int i = 0; i < 100; i++){
        cin >> x;
        if(x <= 10){
            v.emplace_back(i, x);
        }
                 
    }

   for(int i = 0; i < v.size(); i++){
    

        printf("A[%d] = %.1f\n", v[i].first, v[i].second);
   }

}

